server/routes/replyRoutes.ts

```diff
@@ -33,18 +36,21 @@ router.get('/:id', async (req, res, next) => {
 
 })
 
-router.post('/', async (req, res, next) => {
-  try {
-    const reply: Reply = new Reply(req.body.user_id, req.body.post_id, req.body.message);
-    await getRepository(Reply).save(reply);
-    res.status(201).send(reply);
-  } catch (error) {
-    next(Boom.badImplementation());
-  }
-
-})
+router.post('/', validateReply, (req, res, next) => {
+  Promise.all([getRepository(User).findOne(req.body.userId), getRepository(Post).findOne(req.body.postId)]).then(results => {
```

Comment:
You can use array destructuring to get rid of `results[0]` and `results[1]`. Which don't tell that much :D More info here:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring